lib/va_profile/models/disability.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module VAProfile
+  module Models
+    class Disability < Base
+      attribute :combined_service_connected_rating_percentage, String
```

Comment:
@LindseySaari @rjohnson2011 I saw that Ryan used `validates` for the status data here: https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/va_profile/models/veteran_status.rb#L11
Do we also need that?

Reply:
+ to this yes :)

Reply:
hmm... The veteran_status model is also slightly different in that it doesn't have error handling for a `nil` value, like we do in this model:

```ruby
      def self.build_disability_rating(rating)
        return nil unless rating

        VAProfile::Models::Disability.new(
          combined_service_connected_rating_percentage: rating
        )
      end
```
@RachalCassity do you think we still need `validates :combined_service_connected_rating_percentage, presence: true, length: { maximum: 25 }`?

Reply:
Yes

Reply:
All validations should be moved over to the new files

Reply:
I can't find` validates :combined_service_connected_rating_percentage, presence: true, length: { maximum: 25 }` anywhere in master

Reply:
Fixed! 🎉